### PR TITLE
Feature: Implement selectors for blurred background

### LINKED ISSUE #14220: Implement Selectors components with blur style
Implement Selectors components with blur style:

Figma: https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=398%3A18229

The selectors without blur style have already been implemented, so take these components and implement with blur style
- see: https://github.com/status-im/status-mobile/issues/14015

### PR DESCRIPTION
**Pull request by chrispader**
fixes: #14220 

status: ready

### CODE REVIEW

File: src/quo2/screens/selectors/selectors.cljs [hunk 1]
```diff
@@ -29,7 +32,36 @@
                         :disabled? (:disabled? @state)}]
         [text/text {:size :heading-2} "Checkbox Prefill"]
         [quo2/checkbox-prefill {:container-style {:margin-top 0}
-                                :disabled? (:disabled? @state)}]]])))
+                                :disabled? (:disabled? @state)}]]
+
+       [rn/view {:padding-vertical 60
+                 :align-items      :center}
+        [react/blur-view {:style (.-absoluteFill ^js StyleSheet)
```
Comment (ibrkhalil) [inline src/quo2/screens/selectors/selectors.cljs]: Cam we use `oget` ?

File: src/quo2/screens/selectors/selectors.cljs [hunk 2]
```diff
@@ -29,7 +32,36 @@
...
+                          :blurAmount 20
+                          :blurType (if (colors/dark?) :dark :light)}
+         [react/linear-gradient {:style (.-absoluteFill ^js StyleSheet)
```
Comment (ibrkhalil) [inline src/quo2/screens/selectors/selectors.cljs]: Cam we use `oget` ?

**Review by ibrkhalil — APPROVED** [review]
Non blocking comments

File: src/quo2/screens/selectors/selectors.cljs (hunk 1 above)
Comment (chrispader) [reply src/quo2/screens/selectors/selectors.cljs]: sure

**Pull request merged**